v4/source/files.yaml

```diff
@@ -28,6 +28,12 @@
           required: false
           allowEmptyValue: true
           type: string
+        - name: encoded_filename
+          in: formData
+          description: An optional bool string that tells the server if the filename was encoded before upload so that it can be decoded before saving
```

Comment:
In the description can we add a note for the minimum server version this will be included in? Since we don't version the reference with each server release

Just a note like this `**Minimum server version: 4.7**` (assuming it'll be in for 4.7)

Reply:
Markdown doesn't seem to be supported. I simply added the message